exporter/stackdriverexporter/go.mod

```diff
@@ -4,14 +4,16 @@ go 1.14
 
 require (
 	contrib.go.opencensus.io/exporter/stackdriver v0.13.1
+	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v0.2.1-0.20200710010320-8556f4b5bc41
 	github.com/census-instrumentation/opencensus-proto v0.2.1
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
 	github.com/shirou/gopsutil v2.20.4+incompatible // indirect
 	github.com/stretchr/testify v1.6.1
 	go.opencensus.io v0.22.3
 	go.opentelemetry.io/collector v0.5.1-0.20200721173458-f10fbf228f0e
```

Comment:
For some reason my test is failing with this version, but passes when I change it to `go.opentelemetry.io/collector v0.6.0`

Reply:
Likely related to https://github.com/open-telemetry/opentelemetry-collector/pull/1308

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/1308#discussion_r457895403
This is likely because of update to `github.com/golang/protobuf v1.4.2`
Try replacing `assert.Equal` with `proto.Equal` or `cmp.Diff` as in the PR linked above.

Reply:
I think the test is failing because of the `go.opentelemetry.io/collector` version, not the `github.com/golang/protobuf` version. The test still fails even if I remove all uses of `assert.Equal`. Right now, it seems like `BatchWriteSpans` is never even being called. I'm seeing the error message
```
Failed to export to Stackdriver: rpc error: code = Canceled desc = context canceled
--- FAIL: TestStackdriverExport (10.00s)
    stackdriver_test.go:105: test timed out
```
Before I merged master into my branch, I was using
[this version](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/486/commits/b4f771c5b1cb7083766f8de56a55fbcaea8dbf3c#diff-37aff102a57d3d7b797f152915a6dc16R42) of the collector and I didn't experience this issue, and similarly when I change the version back to `0.5.0` or `0.6.0`, the test passes again.

Reply:
Please check again once #503 is merged.